### ISSUE TITLE
Radio Group is not visible in Envelope

### ISSUE THREAD

**Issue by pushp1010** [docusign/native-ios-sdk#108]
@ashokds @mmohareb 

Now we are able to access  radioGroupTabs via below code `recipient.tabs.radioGroupTabs` but this issue still exists as DSMEnvelopeTab.builder does not has radioGroup type expose outside to that we can create a radio group using the below code

```
private func prepareRadioTabs(recipient: DSMSigner) -> [DSMEnvelopeTab] {
            var envelopeTabs: [DSMEnvelopeTab] = []
            var radios: [DSMEnvelopeTab] = []

            recipient.tabs.radioGroupTabs.forEach { group in
                group.radios.forEach { radioTab in
                    // Create RadioTab
                    let frame = radioTab.frameAsCGRect()
                    let grouptab = DSMTabBuilder(for: .radio)
                        .addFrame(frame)
                        .addDocumentId(documentId)
                        .addPageNumber(pageNumber)
                        .addName(value)
                        .addRecipientId(recipientId)
                        .addValue(value)
                        .isLocked(false)
                        .build()
                    envelopeTabs.append(grouptab)
                    radios.append(tab)
                }
               // Create RadioGroup
                let grouptab = DSMTabBuilder(for: .radioGroup)
                    .addFrame(frame)
                    .addDocumentId(documentId)
                    .addPageNumber(pageNumber)
                    .addName(value)
                    .addRecipientId(recipientId)
                    .addValue(value)
                    .isLocked(false)
                    .build()
                envelopeTabs.append(grouptab)
            }
            return envelopeTabs
        }
```

Below is the screenshot of the type expose outside in which radiogroups is still missing. Please help us with the sample code so that we can achieve the same

<img width="449" alt="Screenshot 2021-11-23 at 7 20 02 PM" src="https://user-images.githubusercontent.com/52121407/143036387-0a24e530-c9b7-4081-b454-49311d1d7dd0.png">

_Originally posted by @pushp1010 in https://github.com/docusign/native-ios-sdk/issues/103#issuecomment-976567509_

**Comment by mmohareb** [comment]
@pushp1010  we were able to reproduce this issue on our side, and we are working on a fix, will let you know once we have a timeline when will this be ready.

**Comment by mmohareb** [comment]
Please check the fix for this issue in [SDK version 2.8.0](https://github.com/docusign/native-ios-sdk/releases/tag/v2.8) 

Here is a sample for the usage to add a Radio Buttons Grouped Together

```
        let recipientId: String = "1"
        
        let tabBuilder = DSMTabBuilder(for: .signHere)
        tabBuilder.addName("Signature")
        tabBuilder.addDocumentId("1")
        
        let radioTab = DSMTabBuilder(for: .radio)
            .addName("Radio Button 1")
            .addRecipientId(recipientId)
            .addPageNumber(1)
            .addDocumentId("1")
            .addFrame(CGRect(x: 0, y: 0, width: 100, height: 50))
            .addGroupName("Radio test Group")
            .addTabLabel("Radio 1")
            .addValue("Radio 1")
            .build()
        
        let radioTab2 = DSMTabBuilder(for: .radio)
            .addName("Radio Button 2")
            .addRecipientId(recipientId)
            .addPageNumber(1)
            .addDocumentId("1")
            .addFrame(CGRect(x: 0, y: 100, width: 100, height: 50))
            .addGroupName("Radio test Group")
            .addTabLabel("Radio 2")
            .addValue("Radio 2")
            .build()
        
        tabBuilder.addRecipientId(recipientId)
        tabBuilder.addFrame(CGRect(x: 100, y: 100, width: 50, height: 50))
        tabBuilder.addPageNumber(1)
        
        let recipientBuilder = DSMRecipientBuilder(for: .signer)
        let recipientEmail: String = Keychain.value(forKey: KeychainKeys.accountEmail)!
        let signerName: String = Keychain.value(forKey: KeychainKeys.signerName)!
        recipientBuilder.addSignerEmail(recipientEmail)
        recipientBuilder.addRecipientId(recipientId)
        recipientBuilder.addSignerName(signerName)
        recipientBuilder.add(tabBuilder.build()).add(radioTab).add(radioTab2)
        
        let documentBuilder = DSMDocumentBuilder()
        documentBuilder.addName("Document")
        documentBuilder.addDocumentId("1")
        documentBuilder.addFilePath(Bundle.main.path(forResource: "Portfolio", ofType: "pdf")!)
        
        let envelopeBuilder = DSMEnvelopeBuilder()
        envelopeBuilder.add(recipientBuilder.build())
        envelopeBuilder.addEnvelopeName("DocuSign")
        envelopeBuilder.addEmailSubject("DocuSign.pdf")
        envelopeBuilder.addEmailMessage("Hi, I'm sending you a document to sign and return, ....")
        envelopeBuilder.add(documentBuilder.build())
        
        let envelope = envelopeBuilder.build()
```
